packages/playground/src/components/DistBuilder.res

```diff
@@ -97,7 +97,7 @@ module DemoDist = {
       <div>
         {switch options {
         | Some(options) =>
-          let inputs1 = ProgramEvaluator.Inputs.make(
+          let inputs1 = ForetoldAppSquiggle.ProgramEvaluator.Inputs.make(
```

Comment:
Was this name autogenerated, from "@foretold-app/squiggle"? 

I imagine we should move the codebase to different organization (QURI, for example)